app/containers/Forms/NaicsField.tsx

```diff
@@ -0,0 +1,62 @@
+import {FieldProps} from '@rjsf/core';
+import React from 'react';
+import {Alert} from 'react-bootstrap';
+import {createFragmentContainer, graphql} from 'react-relay';
+import {NaicsField_query} from '__generated__/NaicsField_query.graphql';
+
+interface Props extends FieldProps<string> {
+  query: NaicsField_query;
+}
+
+export const NaicsFieldComponent: React.FunctionComponent<Props> = (props) => {
+  const naicsList = props.query.naicsCodes.edges.map(
+    (edge) => edge.node.naicsCode
+  );
+
+  const fieldProps = {
+    ...props,
```

Comment:
the `query` prop doesn't need to be passed down to the field.
```suggestion
    ...props,
    query: undefined,
```